core/src/main/java/org/eclipse/hono/cache/CacheProvider.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 1.0 which is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ */
+
+package org.eclipse.hono.cache;
+
+/**
+ * A provider for cache instances.
+ */
+public interface CacheProvider {
+
+    /**
+     * Get a new instance of a cache by name. <br>
```

Comment:
please use a `<p>` to separate the first sentence from the remaining description
we also use a verb in *active* form at the start of the first sentence:
*Gets a new instance of a cache by name.*